app/src/main/java/com/mashup/WebConstants.kt

```diff
@@ -0,0 +1,13 @@
+package com.mashup
+
+class WebConstants {
+
+    object URL {
+        const val FACEBOOK = "https://www.facebook.com/mashupgroup/"
+        const val INSTAGRAM = "https://www.instagram.com/official_mashup_/"
+        const val TISTORY = "https://mash-up.tistory.com/"
+        const val YOUTUBE = "https://mash-up.tistory.com/"
+        const val MASHUP_UP_HOME = "https://mash-up.kr/"
+        const val MASHUP_UP_RECRUIT = "https://recruit.mash-up.kr/"
+    }
+}
```

Comment:
calss안에 object 있는 것이 어색한 것 같아요!
```suggestion
object WebUrlContanst {
        const val FACEBOOK = "https://www.facebook.com/mashupgroup/"
        const val INSTAGRAM = "https://www.instagram.com/official_mashup_/"
        const val TISTORY = "https://mash-up.tistory.com/"
        const val YOUTUBE = "https://mash-up.tistory.com/"
        const val MASHUP_UP_HOME = "https://mash-up.kr/"
        const val MASHUP_UP_RECRUIT = "https://recruit.mash-up.kr/"
    }
}
```